docker/koord-runtimeproxy.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.17 as builder
```

Comment:
Do we need the dockerfile for koord-runtime-proxy?  Maybe not.

Reply:
is it inconvenient to install runtime-proxy by souce code or binary?

Reply:
but if install it by container, how to resolve the dependency with kubelet?

Reply:
dependency with kubelet can be mounted by hostpath.